core/src/main/scala/com/clairvoyant/restonomer/core/http/RestonomerRequestBuilder.scala

```diff
@@ -15,6 +15,11 @@ case class RestonomerRequestBuilder(httpRequest: Request[Either[String, String],
   def withHeaders(headers: Map[String, String]): RestonomerRequestBuilder = {
     copy(httpRequest = httpRequest.headers(headers))
   }
+  
+  //Rest - 51
+  def withBody(body: Option[String] = None): RestonomerRequestBuilder= {
+    copy(httpRequest = httpRequest.body(body))
```

Comment:
We cannot pass the Option[String] to the `httpRequest.body()` method as that method accepts only String and not Option[String].